### PR TITLE
 Tabs styles Added 

### PR DESCRIPTION
**Pull request by abhilash-071**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--{repo}--{owner}.hlx.live/
- After: https://feature-metafox-743-tabs--metafox-sites--bmw-importer.hlx.page/test-pages/tabs-comp

### CODE REVIEW

File: blocks/content-navigation/content-navigation.js [hunk 1]
```diff
@@ -272,4 +272,6 @@ export default function decorate(block) {
   scrollRight();
   setTimeout(() => { checkAnchorLinkOverflow(ul, wrapper); }, 400);
   window.addEventListener('hashchange', scrollToHash);
+
+  console.log('hello');
```
Comment (manish11665313) [inline blocks/content-navigation/content-navigation.js:276]: please remove this console log

File: blocks/header/header.js [hunk 2]
```diff
@@ -126,6 +126,26 @@ function toggleMenu(nav, navSections, forceExpanded = null) {
  * decorates the header, mainly the nav
  * @param {Element} block The header block element
  */
+
+// function generateTabsView() {
```
Comment (manish11665313) [inline blocks/header/header.js:130]: please remove this commented code

File: blocks/tabs/tabs.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  });
+    // move $ul below section div
+  $block.replaceChildren($ul);
+
+  // search referenced sections and move them inside the tab-container
+  const $sections = document.querySelectorAll('[data-tab-content-navigation="true"]');
+
+  // move the tab's sections before the tab riders.
+  [...$sections].forEach(($tabContent) => {
+    const name = $tabContent.dataset.anchorId;
```
Comment (manish11665313) [inline blocks/tabs/tabs.js:35]: please add anchorlabel instead of anchorid

**Review by bharath-yathiraj — APPROVED** [review]
approved

**Review by AmolYPatil — APPROVED** [review]
Approved

**Pull request merged**